### ISSUE TITLE
Bug: [prefer-literal-enum-member] Report message should reflect bitwise expression option

### ISSUE THREAD

**Issue by kirkwaiblinger** [typescript-eslint/typescript-eslint#10090]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.
- [X] I have [searched for related issues](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aissue+label%3A%22package%3A+eslint-plugin%22) and found none that matched my issue.
- [X] I have [read the FAQ](https://typescript-eslint.io/linting/troubleshooting) and my problem is not listed.

### Playground Link

https://typescript-eslint.io/play/#ts=5.2.2&showAST=es&fileType=.tsx&code=KYOwrgtgBAYg9nKBvAUFKBBKBeKBGKAanxQF8g&eslintrc=N4KABGBEBOCuA2BTAzpAXGUEKQAIBcBPABxQGNoBLY-AWhXkoDt8B6Y6RAM0Wlsfy8AhvHpNYAW1oTEEgEa90YANrhsOAO5DoTSABowa9VnXZII%2BAHsNAIUr4NlZIgCiADw4pklS01QZ8OEQjbABfEIBdNXDQoA&tsconfig=N4KABGBEDGD2C2AHAlgGwKYCcDyiAuysAdgM6QBcYoEEkJemy0eAcgK6qoDCAFutAGsylBm3TgwAXxCSgA&tokens=false

### Repro Code

```TypeScript
enum Foo {
  A = 1 + 1
}
```

### ESLint Config

```javascript
module.exports = {
  "rules": {
    "@typescript-eslint/prefer-literal-enum-member": [
      "warn", 
      {
        "allowBitwiseExpressions": true
      }
    ]
  }
}
```

### tsconfig

```jsonc
{
  "compilerOptions": {
    "strictNullChecks": true
  }
}
```

### Expected Result

I expect that the rule violation should say something like "Explicit enum value must only be a literal value (string or number), or a bitwise computation" when the `allowBitwiseExpressions` option is enabled.

### Actual Result

"Explicit enum value must only be a literal value (string or number)."
